android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -33,12 +33,12 @@ public MusicControlNotification(ReactApplicationContext context) {
         if(smallIcon == 0) smallIcon = r.getIdentifier("play", "drawable", packageName);
     }
 
-    public void updateActions(long mask) {
+    public void updateActions(long mask, boolean useSkip) {
         play = createAction("play", "Play", mask, PlaybackStateCompat.ACTION_PLAY, play);
         pause = createAction("pause", "Pause", mask, PlaybackStateCompat.ACTION_PAUSE, pause);
         stop = createAction("stop", "Stop", mask, PlaybackStateCompat.ACTION_STOP, stop);
-        next = createAction("next", "Next", mask, PlaybackStateCompat.ACTION_SKIP_TO_NEXT, next);
-        previous = createAction("previous", "Previous", mask, PlaybackStateCompat.ACTION_SKIP_TO_PREVIOUS, previous);
+        next = createAction(!useSkip ? "next" : "skip_forward", "Next", mask, PlaybackStateCompat.ACTION_SKIP_TO_NEXT, next);
+        previous = createAction(!useSkip ? "previous" : "skip_backward", "Previous", mask, PlaybackStateCompat.ACTION_SKIP_TO_PREVIOUS, previous);
```

Comment:
This would not always work, as the action would be already created previously.
Is there really a reason to add rewind/forward to the notification? Can't it just be available to external devices?